src/logger.ts

```diff
@@ -0,0 +1,24 @@
+import { LoggingWinston } from "@google-cloud/logging-winston";
+import winston, { format } from "winston";
+
+export default function getLogger(rateFeed: string): winston.Logger {
+  const isCloudFunction = process.env.FUNCTION_TARGET !== undefined;
+  const transports: winston.transport[] = [
+    // Cloud Logging transport
+    new LoggingWinston({ labels: { rateFeed }, prefix: rateFeed }),
```

Comment:
@chapati23 The `@google-cloud/logging-winston` package has a nice parameter to add a prefix to each log, so we end up with `[$ratefeed] $msg` on GCP. 

However the console transport does not, but I don't think this matters a much when testing locally

Reply:
i updated the local transport to also show the rate feed as a prefix similarly to google cloud